examples/rust-gcoap/Makefile

```diff
@@ -30,6 +34,12 @@ DEVELHELP ?= 1
 # Change this to 0 show compiler invocation lines by default:
 QUIET ?= 1
 
+# Add 3k extra stack: The Rust examples take more of it than gcoap expects,
+# presumably because the example use the standard library's sting formatting
+# instead of one of the more optimized formatters.
+CFLAGS += -DGCOAP_STACK_SIZE='(THREAD_STACKSIZE_DEFAULT + DEBUG_EXTRA_STACKSIZE \
+    + sizeof(coap_pkt_t) + 3072)'
```

Comment:
Because it is incrementing the GCoAP stack size that otherwise defaults to what is now

```
#define GCOAP_STACK_SIZE (THREAD_STACKSIZE_DEFAULT + DEBUG_EXTRA_STACKSIZE \
                          + sizeof(coap_pkt_t) + GCOAP_DTLS_EXTRA_STACKSIZE \
                          + GCOAP_VFS_EXTRA_STACKSIZE)
```

and has no other tunable parameters through which I could add something extra. I can just remove it (replacing it with a constant), or change GCOAP_STACK_SIZE to include a default-0 GCOAP_STACK_SIZE_EXTRA summand.